streamer/src/quic.rs

```diff
@@ -37,8 +37,8 @@ pub const DEFAULT_MAX_STAKED_CONNECTIONS: usize = 2000;
 
 pub const DEFAULT_MAX_UNSTAKED_CONNECTIONS: usize = 500;
 
-/// Limit to 250K PPS
-pub const DEFAULT_MAX_STREAMS_PER_MS: u64 = 250;
+/// Limit to 500K PPS
+pub const DEFAULT_MAX_STREAMS_PER_MS: u64 = 500;
```

Comment:
this is the only functional change. Everything else is tests/comments

Reply:
See https://www.validators.app/validators/mds4GEuiSgQRqveGyktWpETBFCb4AS2wDnhqwLHcT6Z?locale=en&network=mainnet for skip rate info of mds4.

Reply:
So the latest bump is due to the restart? @bw-solana

Reply:
yes, took it offline at ~16:18 UTC